src/content/docs/contributing/modules.md

```diff
@@ -761,6 +761,25 @@ The key words "MUST", "MUST NOT", "SHOULD", etc. are to be interpreted as descri
     All reported versions MUST be without a leading `v` or similar (i.e. must start with a numeric character), or for
     unversioned software, a Git SHA commit id (40 character hexadecimal string).
 
+    <details>
+    <summary>Tips for extracting the version string</summary>
+    `sed` is a powerful stream editor that can be used to manipulate the input text into the desired output.
+    Start by piping the output of the version command to `sed` and try to select the line with the version number:
+
+    ```bash
+    tool --version | sed '1!d'
+    ```
+
+    - `sed '1!d'` Extracts only line 1 of the output printed by `tools --version`.
+    - If the line extraction doesn't work, then it's likely the version information is written to stderr, rather than stdout.
+      In this case capture stderr using `|&` which is shorthand for `2>&1 |`.
+    - `sed 's/pattern/replacement/'` can be used to remove parts of a string. `.` matches any character, `+` matches 1 or more times.
+    - You can separate `sed` commands using `;`. Often the pattern : `sed filter line ; replace string` is enough to get the version number.
+    - It is not necessary to use `echo`, `head`, `tail`, or `grep`.
+    - Use `|| true` for tools that exit with a non-zero error code: `command --version || true` or `command --version | sed ... || true`.
```

Comment:
```suggestion
    - Use `|| true` for tools that exit with a non-zero error code: `command --version || true{:bash}` or `command --version | sed ... || true{:bash}`.
```
for inline code highlighting: https://rehype-pretty-code.netlify.app/#highlight-inline-code